Rabbit/Rabbit/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace Rabbit
+{
+    public enum EyeColors { Blue, Red, Black};
+    public enum FurColors { White, Black, Grey};
+    public enum Gender { Male, Female};
+
+    public class Mammal
+    {
+        public void Characteristics()
+        {
+            Console.WriteLine("I am a mammal and i jump all over the place, a have a lot of speed. I sleep on the grass and i eat carrots.");
+        }
+    }
+
+    public class Rabbit: Mammal
+    {
+        private string eyes;
+        private string fur;
+        private string gender;
+        private DateTime dateofbirth;
+
+        public Rabbit(string eyes, string fur, string gender, string dateofbirth)
```

Comment:
nu stringuri!, daca ai enum, de ce nu le ceri ca si parametru in constuctor? de ce sa faci, equals?